### PR TITLE
Created basic homepage and setup theme

### PR DESCRIPTION
**Pull request by kumsuzM**
### Additions
- Deleted creact-react-app boiler plate (as well as some other random files) and added basic HomePage component to App component.
- Added pages directory which houses HomePage and future pages of application.
- Added a theme in index.tsx for primary and secondary colours (still need to figure out how to use this properly though)/

### Changes
- Made slight changes to the sign-in and sign-out buttons, but these were only stylistic - changed the variant from outlined to contained and added some styling for the margin.

### CODE REVIEW

File: frontend/src/components/SignOutButton.tsx [hunk 1]
```diff
@@ -18,6 +18,8 @@ export const SignOutButton = () => {
   return (
     <Button
       variant="outlined"
+      size="large"
+      color="secondary"
       className="ml-auto"
       onClick={() => handleLogout(instance)}
     >
```
Comment (Siyer2) [inline frontend/src/components/SignOutButton.tsx]: Don't worry too much about the sign out button, I think we'll put that in the nav bar in the chats page

File: frontend/src/components/SignInButton.tsx [hunk 2]
```diff
@@ -24,8 +24,11 @@ export const SignInButton = () => {
...
   return (
     <Button
-      variant="outlined"
+      variant="contained"
       className="ml-auto"
+      size="large"
+      sx={{ mx: 'auto' }}
+      color="secondary"
       onClick={() => handleLogin(instance)}
       disabled={isLoading}
```
Comment (Siyer2) [inline frontend/src/components/SignInButton.tsx:33]: On mobile it seems that this comes outside of the blue background. You can see what this looks like on mobile by opening developer tools (hit F12). And then hit this button on the left
Screen Shot 2022-10-10 at 5 41 04 pm

Feel free to change any of the code I've written in components

**Review by Siyer2 — CHANGES_REQUESTED** [review]
Largely it looks good. A few other things I noticed:

1. To make sure the sign in button still works, you'll need to change the loginPopup to a loginRedirect [here](https://github.com/Siyer2/uni-connector/blob/a4a424e60846e32984c51e41d493c13fdfbede6d/frontend/src/components/SignInButton.tsx#L17)
2. You'll need to do the same thing to make sure sign out works too ([here](https://github.com/Siyer2/uni-connector/blob/a4a424e60846e32984c51e41d493c13fdfbede6d/frontend/src/components/SignInButton.tsx#L17))

File: frontend/src/components/SignInButton.tsx (hunk 2 above)
Comment (kumsuzM) [reply frontend/src/components/SignInButton.tsx:33]: Ah damn. Will look to fix that.

File: frontend/src/theme.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { createTheme } from '@mui/material/styles';
```
Comment (Siyer2) [inline frontend/src/theme.ts:1]: Love that you've moved the theme into it's own file

File: frontend/src/components/SignOutButton.tsx [hunk 4]
```diff
@@ -18,8 +17,6 @@ export const SignOutButton = () => {
   return (
     <Button
       variant="outlined"
-      size="large"
-      color="secondary"
       className="ml-auto"
       onClick={() => handleLogout(instance)}
     >
```
Comment (Siyer2) [inline frontend/src/components/SignOutButton.tsx:25]: Just changed this to match the sign in button. I know we're going to override and move it elsewhere but just want to make sure it's still visible for debugging purposes.

**Review by Siyer2 — APPROVED** [review]
Awesome stuff!

**Pull request merged**